client/src/App.js

```diff
@@ -54,6 +55,7 @@ const App = () => {
           path={ROUTES.HOME}
           render={props => <Home {...props} place={place} />}
```

Comment:
spreading in props like this is actually what is considered an anti-pattern due to the fact that you have no clue what props are being passed. It is a lot more helpful to anyone who comes after you to know as much about the component as possible without having to look directly into the component.